src/modules/Instabug.ts

```diff
@@ -64,7 +64,9 @@ export const init = (config: InstabugConfig) => {
   captureUnhandledRejections();
 
   // Default networkInterceptionMode to JavaScript
-  config.networkInterceptionMode ??= NetworkInterceptionMode.javascript;
+  if (!config.networkInterceptionMode) {
```

Comment:
Let's change this if condition to explicitly check for `null` or `undefined` instead of `!config.networkInterceptionMode` since the enum values will get mapped to integer values with the first enum value being assigned the number `0` so if a user sets the `networkInterceptionMode` to the first enum value JavaScript will treat it as `false` thus changing the user's intended value to the default one even when it's actually set in the user's code.

In this case the first enum value is `javascript` which is the current default so it won't cause bugs, but if we changed the default to be something else users won't be able to set the interception mode to the first enum value as explained above.

```suggestion
  // The `== null` here acts as `=== null or === undefined`
  if (config.networkInterceptionMode == null) {
```